mountpoint-s3/src/main.rs

```diff
@@ -149,6 +149,13 @@ struct CliArgs {
     #[clap(long, help = "Set the 'x-amz-request-payer' to 'requester' on S3 requests", help_heading = BUCKET_OPTIONS_HEADER)]
     pub requester_pays: bool,
 
+    #[clap(
+        long,
+        help = "Mount the filesystem in read-only mode",
```

Comment:
nit: for consistency
```suggestion
        help = "Mount file system in read-only mode",
```